test/test_ops_gradients.py

```diff
@@ -44,7 +45,10 @@ def is_inplace(variant):
             return variant is op.get_inplace()
 
         include_conjugated_inputs = op.test_conjugated_samples and dtype.is_complex
-        samples = op.sample_inputs(device, dtype, requires_grad=True, include_conjugated_inputs=include_conjugated_inputs)
+        small_inputs_only = ((check == "bwgrad_bwgrad" or check == "fwgrad_bwgrad") and
```

Comment:
Instead of adding a new kwarg what about just shrinking the sample inputs?

Reply:
That sounds fine, too

Reply:
@mruberry  Looks like it was timing out again because I removed the reduced input size changes for unary ufuncs. I added it back along with reduced input size changes for binary ufuncs as well (see latest update). Alternatively, what do you think about always skipping slow gradcheck tests for unary/binary ufuncs actually because this should be a case fast gradcheck will always catch.